js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -152,6 +154,7 @@ export const LaunchAssetExecutionButton: React.FC<{
 
 function stateForLaunchingAssets(
   assets: LaunchAssetExecutionAssetNodeFragment[],
+  mouseEvent: React.MouseEvent<any>,
```

Comment:
I'd avoid passing the mouse event any further than the immediate handler function, and instead use a `forceLaunchpad` argument here or something similar based on the `shiftKey` value determined in the handler.

Reply:
Good call, done